src/topology/metric_space/closeds.lean

```diff
@@ -377,7 +377,7 @@ begin
       -- we have proved that `d` is a good approximation of `t` as requested
       exact ⟨d, ‹d ∈ v›, Dtc⟩ },
   end,
-  apply uniform_space.second_countable_of_separable,
+  exact uniform_space.second_countable_of_separable (nonempty_compacts α),
```

Comment:
Is this change necessary?

Reply:
No, I just think that `exact` should always be preferred over `apply`. I can revert it if you don't like it.

Reply:
No need to revert, I just wanted to understand.